frontend/packages/modelServing/modelRegistry/useDeployButtonState.ts

```diff
@@ -0,0 +1,40 @@
+import { SupportedArea } from '@odh-dashboard/internal/concepts/areas/types';
+import useIsAreaAvailable from '@odh-dashboard/internal/concepts/areas/useIsAreaAvailable';
+import { DEPLOY_BUTTON_TOOLTIP } from '@odh-dashboard/internal/pages/modelServing/screens/const';
+import { useAvailableClusterPlatforms } from '../src/concepts/useAvailableClusterPlatforms';
+
+const KSERVE_ID = 'kserve';
```

Comment:
Mentions of KServe under the modelServing package still isn't ideal (I guess at the moment model registry is specific to KServe). However since this is temporary and only MR will be consuming this I'm fine with it.